src/databases/MFIXCDF/avtMFIXCDFFileFormat.C

```diff
@@ -1449,14 +1448,8 @@ avtMFIXCDFFileFormat::GetVectorVar(int domain, const char *varname)
         nzvals= (widths[2]+3);
     }
 
-    // This trick with auto_ptr makes xvec, yvec and zvec get deleted
```

Comment:
Have we lost the auto-deletion since we're no longer using auto_ptr?

Reply:
The original use of `auto_ptr` here was actually incorrect. Any local STL object gets deleted upon going out of scope. So, just normal STL vectors are fine here. Note that the contents of these vectors are later copied to their intended VTK data array destination and it is *that* object that gets returned to caller.